packages/editor/src/lib/editor/Editor.ts

```diff
@@ -8531,6 +8536,7 @@ export class Editor extends EventEmitter<TLEventMap> {
 								(this.getInstanceState().isCoarsePointer ? COARSE_DRAG_DISTANCE : DRAG_DISTANCE) /
 									this.getZoomLevel()
 						) {
+							clearTimeout(this._longPressTimeout)
```

Comment:
(as an aside: i still am thinking about how we prbly want a timeout tracker - otherwise you'll sometimes have some uncollected timeout's if we dispose the editor)

Reply:
That's a good idea!